### ISSUE TITLE
Documentation on how to use theme switcher

### ISSUE THREAD

**Issue by snoepkast** [fish-shell/oh-my-fish#480]
I just installed oh my fish after coming from oh-my-zsh and my main issue was that I wanted to try some themes. According to the docs on the home of the repo I should use the theme switcher, hower that only has robbyrussel available.

Do I have to put every theme in mij config file and `omf install` every time to be able to select it in the theme switcher? Or how does that work?

Also using the theme switcher results in warnings/errors:
![screen shot 2015-06-09 at 15 47 49](https://cloud.githubusercontent.com/assets/1407794/8059191/ef236c04-0ebe-11e5-98f2-4265f36ad912.png)

**Comment by geraintpalmer** [comment]
I am also not sure what to do about this. Listening in, in case something helpful comes back. I've seen the theme switcher work on another person's machine and it's brilliant... :'( :+1:

**Comment by bucaran** [comment]
@geraintpalmer Thanks :)

**Comment by bobthecow** [comment]
I don't believe the theme switcher is `omf install` aware yet. But it _is_ open source if you've got a few minutes and want to tackle the update ;)

**Comment by bucaran** [comment]
@bobthecow Oh, I had no idea.

**Comment by drvinceknight** [comment]
Is there any easy way to download _all_ the themes?

**Comment by drvinceknight** [comment]
This line in the README need to be fixed right?

> Before setting down on a theme, you might want to have a go with all themes using our quick theme switcher by typing theme --help on your shell.

(The hyperlink 404s)

**Comment by nickedes** [comment]
Facing the same problem!  Theme switcher link?

**Comment by Ramblurr** [comment]
Yup, on my old install of oh my fish the themes were all bundled and `theme` could be used to switch them. On a new install, the themes dir is empty, `theme` is gone and `omf list` doesn't list any themes. Installing robbyrussell worked with `omf install robbyrussell` but no other themes are available.

What's up?

**Comment by Ramblurr** [comment]
Ok, I dove into the code. For those interested `omf` is a set of fish functions in `.oh-my-fish/plugins/omf`

The TL;DR is omf is broken right now. 

If you want to install a theme:

1. change it in your ~/.config/fish/config.fish
2. source the file or restart fish
3. run `omf install --theme`

Want a list of themes? Best I could find is https://github.com/oh-my-fish

----

Looking at omf's code I don't see any obvious bugs, rather it does what it is intended, which is to download and install new themes and plugins specified in `config.fish`. It *seems broken* however because in previous versions of oh-my-fish you were able to preview themes and take them for a test drive before editing your config. omf doesn't support this.

My expectation was omf would allow me to discover new themes, but it doesn't.

Any plans to bring back the test drive functionality to oh my fish?

**Comment by bobthecow** [comment]
Yes, this has changed. The `omf` plugin (with the ability to install, manage and update plugins) has only been around for a couple of weeks. Before that, all plugins and themes were part of the main repository, which made it much more difficult to manage. The new way is a lot better for contributors. Unfortunately, the `theme` plugin hasn't been updated to work with `omf` and the new way we install themes… it shouldn't be too difficult, it just hasn't been done yet.

**Comment by bpinto** [comment]
Fixed with #512
